dojson/utils.py

```diff
@@ -30,7 +30,7 @@ def int_with_default(value, default):
 
 
 def ignore_value(f):
-    """Remove key for None value.
+    """Remove key for None or empty value value.
```

Comment:
And invenio_query_parser?

But seriously: this is not a technical question, but rather philosophical.

Are we really delivering a service to the dojson client dealing with bibliographic metadata in preserving empty strings? Or are we adding only confusion?

Maybe I am just wrong and MARC21 really gives importance to empty strings, but in my ten years of experience running multiple document servers I have seen that empty strings were just a source of trubles for final users, cataloguers and in particular developers who always assumed to have non empty strings and forgot to treat the special '' situation.